src/components/HomepageFeatures/index.tsx

```diff
@@ -172,6 +172,50 @@ const FeatureList = [
       </>
     ),
   },
+  {
+    title: <Translate id="index.getting-started.title">Contributor</Translate>,
```

Comment:
This additional `FeatureList` entry should be unnecessary, because all this is covered in the `ContributionFeature` component.

It will also break the layout, because the other entries are 3-column-layouts, whereas this row is now changed to a 2-column-2-row-layout.